gpiozero/boards.py

```diff
@@ -2063,15 +2065,15 @@ def __init__(self, socket=None, initial_value=False, pin_factory=None):
             raise EnergenieSocketMissing('socket number must be provided')
         if not (1 <= socket <= 4):
             raise EnergenieBadSocket('socket number must be between 1 and 4')
-        if initial_value is None:
-            raise EnergenieBadInitialValue("initial value can't be None")
+        if initial_value not in (True, False, None):
+            raise EnergenieBadInitialValue("initial value must be bool or None")
         self._value = None
         super(Energenie, self).__init__(pin_factory=pin_factory)
         self._socket = socket
         self._master = _EnergenieMaster(pin_factory=pin_factory)
         if initial_value:
             self.on()
-        else:
+        elif initial_value is False:
```

Comment:
I don't think we should be checking explicitly for True or False, but instead simply checking for None, and then just using duck-typing otherwise? See e.g. https://github.com/gpiozero/gpiozero/blob/master/gpiozero/input_devices.py#L112

I think this also means the `if initial_value not in (True, False, None):` line would need to be removed?